### ISSUE TITLE
Providers must specify one or more URL scheme and API endpoint pairs

### ISSUE THREAD

**Issue by Riuujin** [iamcal/oembed#100]
From what I understand this means that all providers must specify these, yet for YouTube I don't see any scheme url's. Am I missing something here or is the http://oembed.com/providers.json simply incomplete?

**Comment by iamcal** [comment]
yeah, it seems like these are just missing. the youtube examples indicate the scheme partially, but i'm not sure what is covered: https://github.com/iamcal/oembed/blob/master/providers.yml

**Comment by gotwig** [comment]
I dont know why this hasnt been fixed for almost 2 freaking years. the list is pretty incomplete.

**Comment by iamcal** [comment]
There's not really any need - most providers (including YouTube) just use the discovery link tags

**Comment by raae** [comment]
If one is to find links in content and replace them with the html from the oEmbed response wouldn't that require a complete list of schemes? As far as I understand WordPress and Drupal keeps their own providers and schemes lists for this purpose. 

Otherwise one would have to call every link one find to get the tag info, and then call the endpoint to get the result? Or am I missing something in my understanding of how this works?

**Comment by raae** [comment]
Would you @iamcal be interested if I made a pull request with fixes to the provider list? I think it would be nice to have a complete list as possible.

**Comment by iamcal** [comment]
Most consumers of oEmbed use discovery, which does not require a registry. While I still merge registry update pull requests, it's not necessary that they be correct.

With that said, I'll happily accept any pull requests that update the provider registry.
